cmake_common_scripts/LICENSE.BSD-3-Clause

```diff
@@ -0,0 +1,30 @@
+BSD 3-Clause License
+
+Copyright (c) 2020, ABB Schweiz AG
+Copyright (c) 2020, Southwest Research Institute
```

Comment:
I'm not sure you should add this. SwRI hasn't added anything.

perhaps the entire "copyright" line shouldn't be here, as you're providing the license text itself, without attaching it to anything specific here.

The attribution would then be close to the snippet you copied.

Reply:
> I'm not sure you should add this. SwRI hasn't added anything.

A few of the function are ones that I had added like configure_package, and those related discover gtest. Just noticed that SwRI copyright was not present.

Reply:
> perhaps the entire "copyright" line shouldn't be here, as you're providing the license text itself, without attaching it to anything specific here.

Is this still relevent breaking out the code into a separate file?

Reply:
I had understood you only added the BSD license file for the snippet we're discussing in https://github.com/ros-industrial/abb_robot_driver/issues/8. If there is more BSD covered code, then of course you should do whatever it necessary.